make-workflows.sh/install.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+set -euo pipefail
```

Comment:
Don't set pipefail because we expect to be able to do things like test if the result of a chain with `grep` and `cut` is empty, and not error immediately.

Reply:
okay accepted.